package.json

```diff
@@ -138,7 +138,8 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
-    "lint": "bin/run-lint"
+    "lint": "bin/run-lint",
+    "css-lint": "stylelint client/**/**/*.scss --syntax scss"
```

Comment:
This should be `stylelint 'client/**/*.scss' --syntax scss`

As written, this `**` is passed to the shell, so it doesn't work as intended and we miss a bunch of files, like `client/auth/style.scss`.  Adding the quotes will cause the pattern to be passed to `node-glob` as intended, see https://github.com/stylelint/stylelint/blob/6.5.1/src/cli.js#L27-L35.